scripts/core/editor3/directive.tsx

```diff
@@ -393,6 +573,13 @@ class Editor3Directive {
                     }
                 });
 
+                // when validation status changes, increment `refreshTrigger` which will cause editor3 to re-render
+                $scope.$watch('vm.validationError', (val, old) => {
+                    if (val !== old) {
+                        this.refreshTrigger++;
```

Comment:
if you do `scope.$applyAsync()` wouldn't that update the editor too?

Reply:
Who knows. I'm continuing to use the same method as already used in this file. I would guess that applyAsync wouldn't help, because it's a parent scope we want to re-render. We'll be removing angular authoring anyway so it's not worth putting time to investigate this.